filament/src/ColorGrading.cpp

```diff
@@ -89,13 +102,13 @@ FColorGrading::FColorGrading(FEngine& engine, const Builder& builder) {
     constexpr size_t elementSize = sizeof(half4);
     void* const data = malloc(lutElementCount * elementSize);
 
-    ToneMapper toneMapper = selectToneMapping(builder->mToneMapping);
+    ToneMapper toneMapper = selectToneMapping(builder->toneMapping);
 
     //auto now = std::chrono::steady_clock::now();
 
     // Multithreadedly generate the tone mapping 3D look-up table using 32 jobs
     // Slices are 8 KiB (128 cache lines) apart.
-    // This takes about 3-6ms on Android in Release mode
+    // This takes about 3-6ms on Android in Release mode on Pixel 4
```

Comment:
Actually this was 3ms on Pixel4 and 6ms on MotoG7, that's why I didn't specify Pixel4.

Reply:
Will remove then.